src/rest-client-v2.ts

```diff
@@ -145,6 +145,23 @@ export class RestClientV2 extends BaseRestClient {
     );
   }
 
+  /**
+*
+* * Common | Assets
+*
+*/
+  getFundingAssets(params?: object): Promise<APIResponse<any>> {
+    return this.getPrivate(`/api/v2/account/funding-assets`, params);
+  }
+
+  getBotAccount(params?: object): Promise<APIResponse<any>> {
+    return this.getPrivate(`/api/v2/account/bot-assets`, params);
+  }
+
+  getAssetsOverview(): Promise<APIResponse<any>> {
```

Comment:
```suggestion
    /** Get assets overview */
    getBalances(): Promise<APIResponse<any>> {
```

I would prefer this kind of endpoint is called get balances. It's more intuitive & consistent with our other SDKs, even if it's slightly different from what the api doc calls it for bitget